resources/views/accounts/_edit_notifications.blade.php

```diff
@@ -28,6 +28,33 @@
                     </div>
                 </label>
             </div>
+
+            <div
+                class="account-edit-entry account-edit-entry--no-label js-account-edit js-user-preferences-update"
```

Comment:
`js-user-preferences-update` causes error because the endpoint doesn't return anything.